sources/packages/web/tsconfig.json

```diff
@@ -10,8 +10,7 @@
     "skipLibCheck": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "sourceMap": true,
-    
+    "sourceMap": true,    
```

Comment:
Avoid including unnecessary files in the review.
Please do not consider this as a blocker for this review.